app/models/species.rb

```diff
@@ -1,4 +1,4 @@
-class Specie < ApplicationRecord
+class Species < ApplicationRecord
```

Comment:
I think the singular name for Species is species....I talk to @JorgeATR about this, maybe he can argue on this...